.github/workflows/main.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [windows-latest, ubuntu-latest, macos-latest]
+        os: [windows-latest]
```

Comment:
I would add all of them

Reply:
I will add them when the project is at a more mature stage, right now the project it's using `pyreadline3` which is a lib available only on Windows based systems.